spec/linux/kernel.go

```diff
@@ -42,5 +43,72 @@ func (g *KernelGenerator) Generate() (interface{}, error) {
 		results[key] = str
 	}
 
+	hostInfo, err := host.Info()
+	if err != nil {
+		kernelLogger.Errorf("Failed to get platform information: %s", err)
+	}
+
+	if np := normalizePlatform(hostInfo.Platform); np != "" {
+		platformInfo := np + " " + hostInfo.PlatformVersion
```

Comment:
[comment]
Once you concatenate platform and version, you cannot get platform and version information separately (np can contains empty string), so it might be useful to  store data separately, and concatenate it when using it in view page or somewhere else.

How do you think?